assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -38,8 +38,8 @@ transparent_hugepages=never
 
 {{if not .GloballyDisableIrqLoadBalancing}}
 [irqbalance]
-#> Override the value set by cpu-partitioning with an empty one
-banned_cpus=""
+#> Disable the plugin entirely. It can be racy if TuneD restarts for whatever reason. We depend now on a systemd unit.
```

Comment:
We can add a comment for future devs about the fact that this plugin is enabled in the parent profile (cpu-partitioning) thus needing to disable it in the child

Reply:
Sure, I will update both PRs.